anyway/models.py

```diff
@@ -1211,24 +1210,6 @@ class SuburbanJunction(Base):
                                            nullable=True)
     roads = Column(postgresql.ARRAY(Integer(), dimensions=1), nullable=False)
 
-    @staticmethod
-    def get_hebrew_name_from_id(non_urban_intersection: int) -> str:
-        res = db.session.query(SuburbanJunction.non_urban_intersection_hebrew).filter(
-            SuburbanJunction.non_urban_intersection == non_urban_intersection).first()
-        if res is None:
-            raise ValueError(f"{non_urban_intersection}: could not find "
-                             f"SuburbanJunction with that symbol")
-        return res.non_urban_intersection_hebrew
-
-    @staticmethod
-    def get_id_from_hebrew_name(non_urban_intersection_hebrew: str) -> int:
-        res = db.session.query(SuburbanJunction.non_urban_intersection).filter(
-            SuburbanJunction.non_urban_intersection == non_urban_intersection_hebrew).first()
-        if res is None:
-            raise ValueError(f"{non_urban_intersection_hebrew}: could not find "
-                             f"SuburbanJunction with that name")
-        return res.non_urban_intersection
-
```

Comment:
@ziv17 are we using these functions?

Reply:
Hi @atalyaalon I assume they will be used when we will add suburban junction resolution.